src/node/rpc/test/stub_consensus.h

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Can this file be replaced by `kv/test/stub_consensus.h`? Or is it doing something different I'm missing?

Reply:
There's at least two things, one is the node_id/node_set is consistent in the new file, but that can probably be fixed. The other is that the new stub has signatures turned off. I'll try to merge them still because they are quite similar, I did not know we had this one.